syntax = "proto3";
package iroha.protocol;
import "schema/primitive.proto";
import "schema/commands.proto";

message Transaction {
    message Payload {
        repeated Command commands = 1;
        string creator_account_id = 2;
        uint64 tx_counter = 3;
        uint64 created_time = 4;
    }

    Payload payload = 1;
    repeated Signature signature = 2;
}